docker-compose.yaml

```diff
@@ -375,6 +361,14 @@ services:
     restart: always
     volumes: ["./docker/karton.ini:/etc/karton/karton.ini", "${DOCKER_COMPOSE_ADDITIONAL_SHARED_DIRECTORY:-./shared}:/shared/"]
 
+  karton-SubdomainEnumeration:
```

Comment:
here karton-subdomain_enumeration, consistent with the rest of the list

btw, kartons should be alphavetically sorted